### DISCUSSION TITLE
I'm suggesting that client can set color for rectangles (for diff, excluded part)

### DISCUSSION THREAD

**Discussion by woosyume** [romankh3/image-comparison#204] [Ideas]
Nice to meet you.
This is Kim who introduced this library on my work.

## What
I'm suggesting that client can set color for rectangles with the difference.

## Background
Our design consists of many red colored features so it's not easy to see where is the diff rectangle.

## How
1. Add a public method to ImageComparison.java to set color.

*Sample*
```
    public Color setDiffRectangleColor(Color color) {
        this.color = color;
        return this;
    }
...
        graphics.setColor(this.color);
```

**Comment by romankh3** [comment]
Hello Kim.
Nice to meet you, too.

I think we can do it.

Feel free to create a feature request with all the needed information for you.

Best regards,
Roman.

**Reply by woosyume** [reply]
Thank you for your quick response.
Let me prepare the pull request.

Regards,
Kim

**Reply by romankh3** [reply]
before you start adding a feature, you need to know few details:
1. create a new issue with this feature to document it.
2. we use GitFlow, so you need to create your branch from `development` and create a pull-request to `development` as well.

**Reply by woosyume** [reply]
Thanks for informing that
I'll follow the rule.

Regards,
Kim

**Comment by romankh3** [comment]
@woosyume could you please describe the case, in which you want to use `image-comparison`? It's needed for me to a deeper understanding of library usage.

**Reply by woosyume** [reply]
@romankh3  
I'm not sure what is definition of the case you're saying but let me explain it with thinking that it's use case.
About my case, 
When we release something, we run image comparison for before and after deployment.
It's job of internal automation app based on Selenium WebDriver.

But we sometime take time to find diff rectangle when it's exists because there are some other parts with red colored.
Maybe you can see it with the following page.
https://www.rakuten.co.jp/

That's why I want to set other color so that we can find difference as soon as possible.

Please feel free to let me know if you need further explanation.
Best Regards,

Kim
